test/scrabble_test.rb

```diff
@@ -6,8 +6,30 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+  
+  def test_it_can_score_upcase_words
```

Comment:
Consider testing `#sum_letter_point_values` directly in order to include some unit tests.